app/javascript/gobierto_data/modules/gobierto_data_controller.js

```diff
@@ -1,6 +1,9 @@
 import Vue from "vue";
 import { router } from "../lib/router";
-import 'codemirror/lib/codemirror.css'
+import 'codemirror/lib/codemirror.css';
+import { checkAndReportAccessibility } from "lib/vue/accesibility";
+
+checkAndReportAccessibility()
```

Comment:
Doing this, we're forcing to check and make the reports every single time anybody surf on the site. Is that what we want? Wouldn't be better wrap into some flag, i.e. `process.env.ACCESSIBILITY`, and only run this code when the developer requires it?

Reply:
I agree with Hugo but in the opposite way, I'd like it to be enabled by default to everybody and just in case you don't want it you could disable it with an env var.